R/estimate_CUMINC.R

```diff
@@ -0,0 +1,89 @@
+#' Competing Events Cumulative Incidence
```

Comment:
@title ....
@description ....

We use roxygen tags for scraping the files to make a validation report easier to create

Reply:
Done! ✔️